content/about/notes/1.1/index.md

```diff
@@ -49,15 +49,24 @@ Istio 1.1.
 
 ## Traffic management
 
-- **New `Sidecar` Resource**. Added support to limit the set of services
-  visible to sidecar proxies in a given namespace using the `Sidecar` resource.
-  This limit reduces the amount of configuration computed and transmitted to
-  the proxy. On large clusters, we recommend adding a sidecar resource per
-  namespace.
-
-- **Restrict Visibility of Networking Resources**. Added the new `exportTo`
-  field to all networking resources which lets you control the visibility of
-  individual resources to specific namespaces.
+- **New [`Sidecar`](/docs/concepts/traffic-management/#sidecars) Resource**. Enables more fine-grained
```

Comment:
links in the subject lead-ins feel a little one-off and strange.  Can you try `**New `Sidecar` Resource**.  Added support via the [sidecar resource](/docs/concepts/traffic-management/#sidecars).  This feature enables...`

Reply:
also links don't need backticks.

Reply:
Done